[34m[1mwandb[39m[22m: [33mWARNING[39m Saving files without folders. If you want to preserve subdirectories pass base_path to wandb.save, i.e. wandb.save("/mnt/folder/file.h5", base_path="/mnt")
Traceback (most recent call last):
  File "f:\Documents\acm\2024\8\13\tem.py", line 5, in <module>
    wandb.save(os.path.abspath(__file__), policy="now")
  File "F:\Software\anaconda3\envs\d2l\Lib\site-packages\wandb\sdk\wandb_run.py", line 400, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Software\anaconda3\envs\d2l\Lib\site-packages\wandb\sdk\wandb_run.py", line 390, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Software\anaconda3\envs\d2l\Lib\site-packages\wandb\sdk\wandb_run.py", line 1981, in save
    return self._save(
           ^^^^^^^^^^^
  File "F:\Software\anaconda3\envs\d2l\Lib\site-packages\wandb\sdk\wandb_run.py", line 2045, in _save
    target_path.symlink_to(source_path)
  File "F:\Software\anaconda3\envs\d2l\Lib\pathlib.py", line 1386, in symlink_to
    os.symlink(target, self, target_is_directory)
OSError: [WinError 1314] 客户端没有所需的特权。: 'f:\\Documents\\acm\\2024\\8\\13\\tem.py' -> 'F:\\Documents\\acm\\wandb\\run-20240814_210240-06l2iqq7\\files\\13\\tem.py'